core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/Display.scala

```diff
@@ -2,19 +2,10 @@ package org.virtuslab.ideprobe.ide.intellij
 
 sealed trait Display
 object Display {
-  private val Default = Native
-
-  val Mode: Display = {
-    Option(System.getenv("IDEPROBE_DISPLAY"))
-      .map(fromName)
-      .getOrElse(Default)
-  }
 
   val XvfbDisplayId = "7" // doesn't really matter as long as it is not 0
 
-  println(s"Display mode: $Mode")
```

Comment:
It is useful information, always printed in the beginning of CI job. Especially helpful if you don't know if it was configured correctly